library/sql.inc

```diff
@@ -61,7 +61,7 @@ if (!$sql_strict_set_success) {
 //  command does not work)
 $GLOBALS['adodb']['db']->SetFetchMode(ADODB_FETCH_ASSOC);
 
-if ($GLOBALS['debug_ssl_mysql_connection']) {
+if (isset($GLOBALS['debug_ssl_mysql_connection']) && $GLOBALS['debug_ssl_mysql_connection']) {
```

Comment:
This particular line :boom:'d when I attempted to use sql.inc in the new "fixture framework". I suppose it's because the app isn't bootstrapping itself within the test "branch" of the project.

Reply:
hi @dixonwhitmire ,
If globals.php isn't included somewhere then lots of things will break. Is that not being included in the test runs?

Reply:
Is this where things are being "bootstrapped" for these tests:
https://github.com/openemr/openemr/blob/master/phpunit.xml#L5
https://github.com/openemr/openemr/blob/master/tests/bootstrap.php

Reply:
If so, I wonder if it would be safe to include globals.php with the `$ignoreAuth = true` there instead; note we would need to be sure of security if do something like that.

Reply:
Thanks @bradymiller! I will take a look to see if I can wire that up. I'll start out by reverting the changes I made to sql.inc and the "event logger" to see how things work out.